internal/interceptor/server_test.go

```diff
@@ -36,7 +36,7 @@ func TestRenderPipeline(t *testing.T) {
 	var odsConfig *config.ODS
 	err := yaml.Unmarshal(conf, &odsConfig)
 	fatalIfErr(t, err)
-	gotPipeline, err := renderPipeline(odsConfig, data, "-v0-1-0")
+	gotPipeline, err := renderPipeline(odsConfig, data, "ClusterTask", "-v0-1-0")
```

Comment:
Would a test with "Task" instead of "ClusterTask" as param test anything different?

Reply:
Hehe, I thought about but think "not really". We could put some gibberish in here but that wouldn't really bring more confidence. In any case, `ClusterTask` is what a production use case should have in place so that's what should be tested.